package.json

```diff
@@ -13,6 +13,8 @@
     "ejs": "~2.3.4",
     "finalhandler": "^0.5.0",
     "lodash": "^3.10.1",
+    "moment": "~2.13.0",
```

Comment:
Another way would be to be able to pass a function to deserialize specific fields by name or by type. Could be in options.

Then the code would look like :


if (customDeserializerExists(name)) {
value = customDeserializerExists(name)(text);
}
else {
 
// current code
if (name === 'int' || name === 'integer') {
      value = parseInt(text, 10);
    } else if (name === 'bool' || name === 'boolean') {
      value = text.toLowerCase() === 'true' || text === '1';
...
...

}